docs/api/dandi-client.md

```diff
@@ -0,0 +1,66 @@
+# DANDI Client
+
+The DANDI Client is a Python library and command-line tool for interacting with the DANDI Archive. It provides functionality for downloading, validating, organizing, and uploading data to and from the DANDI Archive.
+
+## Installation
+
+You can install the DANDI Client using pip:
+
+```bash
+pip install dandi
+```
+
+## Documentation
+
+The full documentation for the DANDI Client is available at [https://dandi.readthedocs.io/](https://dandi.readthedocs.io/).
+
+## Key Features
+
+- Download data from the DANDI Archive
+- Validate NWB files
+- Organize data for upload to the DANDI Archive
+- Upload data to the DANDI Archive
+- Search for Dandisets and assets
+- Manage Dandisets and their metadata
```

Comment:
Can the DANDI Client manage metadata?  I thought Dandiset metadata had to be edited in the web app.

```suggestion
- Manage Dandisets
```

Reply:
You can edit metadata programmatically too: https://dandi.readthedocs.io/en/latest/modref/dandiapi.html#dandi.dandiapi.RemoteDandiset.set_metadata